packages/preview/exam-lecture-zh/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "exam-lecture-zh"

```

Comment:
I feel like this name is a bit too descriptive, which goes against [our naming guidelines](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#naming-rules). Can you change it to something else, please?